modules/apps/portal/portal-cluster-multiple/src/test/java/com/liferay/portal/cluster/multiple/internal/ClusterExecutorImplTest.java

```diff
@@ -66,57 +68,6 @@ public static void setUpClass() {
 		PropsUtil.setProps(new PropsImpl());
 	}
 
-	@Test
-	public void testClusterEventListener() {
```

Comment:
@W2jx Please don't just remove the test case; instead, register mocked event listeners to migrate this test case to cover our change.